### PR TITLE
Create Migrations Directory

### PR DESCRIPTION
**Pull request by rachelwhitton**
Connects to #415 
1 of 3 PRs

This PR: 
- Creates Migrate directory
- URL changes throughout docs
- Creates migrate.md landing page
 - Revises migrate.md so that it is a comprehensive, high level guide through the entire migration process. This doc is agnostic and links to CMS specific docs created in PR's #726 and #727. 
 - This doc replaces the existing "Importing an Existing Site" doc - links updated throughout. 

Before merging this PR, both #726 and #727 must be merged.

### CODE REVIEW

**Comment by rachelwhitton** [comment]
@bmackinney this is the main PR for #415 

Can you review #726 and #727 first and merge them into this feature branch when they have been approved. They are 2 export docs - one for drupal and one for wp. Both docs link to the new migrations landing page created within this PR. 

This PR will have to be tested thoroughly for broken links before final deployment with an accurate list of rewrites.

File: source/docs/articles/sites/migrate.md [hunk 1]
```diff
@@ -43,98 +36,59 @@ When preparing a site for export, there are a few best practices to follow:
 * Take a look at your codebase and **remove any non-core code from your site** that you aren’t planning on running on Pantheon.
 * If you’ve been using the database for things other than Drupal or WordPress, you should **drop or skip any unnecessary or unrelated database tables** that your site doesn’t need.
 
+
 ### Create Single-file Archives
-If your site can be packaged with a total archived size less than 500MB, then you can import that single file to create your site.
-Drupal archives created with [drush archive dump](http://drushcommands.com/drush-6x/archive/archive-dump) are known to work when imported during site creation.
+Site's that can be packaged with a total archived size less than 500MB are able to use single-file archives during the import process. These archives can be created with [drush](/docs/articles/drupal/prepare-drupal-for-export#create-archive-using-drush) for Drupal sites and  [Plugins](/docs/articles/wordpress/export-an-existing-wordpress-site#export-wordpress-via-plugins) for WordPress.
```
Comment (rachelwhitton) [inline source/docs/articles/sites/migrate.md]: @bmackinney I still need to work in what goes into a single-file archive. Do you think that should be here in the main migration doc, or should it be specific to each CMS in their respective export docs?

**Comment by rachelwhitton** [comment]
@ari-gold @bmackinney @newtoid can you review and provide feedback for reorganizing the import docs? 

Connects to #726 and #727

**Comment by rachelwhitton** [comment]
Deleted Docs: 
**Importing an Existing Site**
**Importing a WordPress Site**
## Redirects Needed
- Above docs to /docs/articles/sites/migrate
- /docs/articles/sites/create/migrating-sites → /docs/articles/sites/migrate
- /docs/articles/sites/create/importing-a-large-site → /docs/articles/sites/migrate/importing-a-large-site
- /docs/articles/sites/create/importing-drush-site-archives-with-terminus → /docs/articles/sites/migrate/importing-drush-site-archives-with-terminus

**Comment by rachelwhitton** [comment]
We need to preserve changes made in #744 about import failing due to the use of sessions on WP sites - @bmackinney do you think that should go into the migrate doc, or in the export wp doc?

**Comment by bmackinney** [comment]
@rachelwhitton please edit and pass to CSE for review

**Comment by rachelwhitton** [comment]
CSE - Ready for review.

**Pull request merged**